dvc/command/version.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import unicode_literals
+
+import platform
+import argparse
+import logging
+
+from dvc.command.base import CmdBase, append_doc_link
+from dvc.version import __version__
+
+
+logger = logging.getLogger(__name__)
+
+
+class CmdVersion(CmdBase):
+    def run(self):
+        info = (
+            "DVC version: {dvc_version}\n"
+            "Python version: {python_version}\n"
+            "Platform: {platform_type}"
+        ).format(
+            dvc_version=__version__,
+            python_version=platform.python_version(),
+            platform_type=platform.platform(),
+        )
+        logger.info(info)
+        return 0
+
+
+def add_parser(subparsers, parent_parser):
+    ADD_HELP = (
```

Comment:
Why this constant is named `ADD_HELP` looks like you copy paste it from `add` and forgot to change the name :stuck_out_tongue:

Reply:
@mroutis Oops. Yeah, I copied it from there only. :sweat_smile:. Will change it so that codeclimate's test also passes.